person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, name = 'Unknown', age = 0, parent_permission: true)
+    @id = id
```

Comment:
- [OPTIONAL ]
You can make the id to be obtained randomly
 `@id = Random.rand(1..1000) `